lnrpc/gen_protos.sh

```diff
@@ -20,14 +20,15 @@ function generate() {
       "${file}"
   
     # Generate the REST reverse proxy.
+    annotationsFile=${file//proto/yaml}
     protoc -I/usr/local/include -I. \
-      --grpc-gateway_out=logtostderr=true,paths=source_relative,grpc_api_configuration=rest-annotations.yaml:. \
+      --grpc-gateway_out=logtostderr=true,paths=source_relative,grpc_api_configuration=${annotationsFile}:. \
```

Comment:
I believe you also need to update the `rpc-check` target in the `Makefile`.

Reply:
Ah yes, good catch. Seems a lot of other checks failed as well :sweat_smile: